ai8x.py

```diff
@@ -33,6 +33,60 @@ def __call__(self, img):
         return img.sub(0.5).mul(256.).round().clamp(min=-128, max=127).div(128.)
 
 
+class fold:
```

Comment:
I think we need to CamelCase the class names according to PEP8 format. "normalize" needs to be changed too. 

Surely this is not critical, we can change them later in another commit.

Reply:
I think it is better to fix all in another commit. It requires a big regression test.